src/charactersheet/models/dm/encounter_sections/monster.js

```diff
@@ -60,6 +60,26 @@ export class Monster extends KOModel {
         }
     });
 
+    modifier = (score) => {
```

Comment:
Do we not have any of this math in a common place?

Reply:
We only had it in Player Ability Scores and since Monsters don't have an ability score model on the client-side it wasn't easily portable.